### PR TITLE
Corrected deleteTextRecord code snippet.

### PR DESCRIPTION
**Pull request by austinvach**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ New Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 👷 Example Application
- [ ] 🧑‍💻 Code Snippet
- [ ] 🎨 Design
- [ ] 📖 Content
- [ ] 🧪 Tests
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

The code snippet was missing record.id which was causing a reference error. 

First pull request, pardon any mistakes in process.

## Related Tickets & Documents

Resolves this issue: https://github.com/TBD54566975/developer.tbd.website/issues/1070

## Added code snippets?
- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [x] 🤷‍♂️ kinda, updated an existing one

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [x] 🙋 no, because I need help

### No tests? Add a note

It would seem to be covered by https://github.com/TBD54566975/developer.tbd.website/blob/542507f9f48b9a202ad94309c3b397f57bef61a3/site/__tests__/quickstart.test.js#L51 but I'm not sure why it didn't catch the issue in the first place.

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 general documentation
- [x] 🙅 no documentation needed

### No docs? Add a note

I'm not adding anything new, just correcting what was already there.

## [optional] What gif best describes this PR or how it makes you feel?
First ever pull request...
https://media.giphy.com/media/3ohhwF34cGDoFFhRfy/giphy.gif

### CODE REVIEW

**Comment by angiejones** [comment]
hey @austinvach! congrats on your first-ever PR! We're honored that you chose TBD to contribute to ❤️

Adding `record.id` is optional here, because the variable name is the same as the property name and in JavaScript, you don't have to repeat it. 

```js
export async function deleteTextRecord(web5, did, recordId) {
  const deleteResult = await web5.dwn.records.delete({
    message: {
      recordId,
    },
  });

  return deleteResult;
}
```

However, for documentation, I think it would be good to be as explicit as possible to avoid any confusion. So, I say all of that to say - let's get your fix in!

You'll need to add your fix to [quickstart.js](https://github.com/TBD54566975/developer.tbd.website/blob/main/site/code-snippets/web5/quickstart.js#L37) instead of `code-snippets-map.json`. code-snippets-map.json is automatically generated from the code snippets throughout the site, so your manual change there won't persist.

Let us know if you have any questions! 😊

**Review by angiejones — CHANGES_REQUESTED** [review]
left note to make change in a different file

**Comment by austinvach** [comment]
Done. Thanks for the guidance.

File: site/code-snippets/web5/quickstart.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ export async function updateTextRecord(record) {
 export async function deleteTextRecord(web5, did, recordId) {
   const deleteResult = await web5.dwn.records.delete({
     message: {
-      recordId,
+      recordId: record.id,
```
Comment (angiejones) [inline site/code-snippets/web5/quickstart.js:37]:
```suggestion
      recordId: recordId
```

It needs to use that argument from line 34

File: site/src/util/code-snippets-map.json [hunk 2]
```diff
@@ -53,5 +53,5 @@
   "didCreate": "const { web5, did: aliceDid } = await Web5.connect();",
   "readTextRecord": "const readResult = await record.data.text();",
   "updateTextRecord": "const updateResult = await record.update({\n    data: 'Hello, Web5! I am updated.',\n  });",
-  "deleteTextRecord": "const deleteResult = await web5.dwn.records.delete({\n    message: {\n      recordId,\n    },\n  });"
-}
\ No newline at end of file
+  "deleteTextRecord": "const deleteResult = await web5.dwn.records.delete({\n    message: {\n      recordId: record.id,\n    },\n  });"
```
Comment (angiejones) [inline site/src/util/code-snippets-map.json:56]:
```suggestion
  "deleteTextRecord": "const deleteResult = await web5.dwn.records.delete({\n    message: {\n      recordId: recordId,\n    },\n  });"
```

**Review by angiejones — CHANGES_REQUESTED** [review]

**Review by angiejones — APPROVED** [review]

**Pull request merged**